### PR TITLE
Clean up some codesmells and comments

### PR DESCRIPTION
**Pull request by LeviUniAcc**

### CODE REVIEW

**Review by marlau97 — APPROVED** [review]

File: Assets/Scripts/Weapon/ShootWeapon.cs [hunk 1]
```diff
@@ -26,9 +26,9 @@ void Update()
     /// </summary>
     private void Shoot()
     {
-        if (Input.GetButtonDown("Fire1") && !Global.instance.GetRoundStatus())
+        if (Input.GetButtonDown("Fire1") && !GameManager.instance.GetRoundStatus())
```
Comment (delvh) [inline Assets/Scripts/Weapon/ShootWeapon.cs:29]: Wouldn't it perhaps be a good idea to declare the manager as a variable first?

File: Assets/Scripts/Weapon/ShootWeapon.cs [hunk 2]
```diff
@@ -49,17 +49,17 @@ private void handleKill(RaycastHit hit)
     {
         if (hit.transform.tag == "CorrectAnswer")
```
Comment (delvh) [inline Assets/Scripts/Weapon/ShootWeapon.cs:50]: Wait, you're using string comparison to see if the answer matches?
Is there no other possible approach?
In the worst case, the least you could do is store the string literals as constants.

Comment (delvh) [reply Assets/Scripts/Weapon/ShootWeapon.cs:50]: Feared so.
Then at least compare constants, and not string literals.

Comment (LeviUniAcc) [reply Assets/Scripts/Weapon/ShootWeapon.cs]: done

**Review by delvh — APPROVED** [review]

**Pull request merged**